pkg/dockercreds/access_checker.go

```diff
@@ -60,21 +64,3 @@ func HasWriteAccess(tagName string) (bool, error) {
 
 	return true, nil
 }
-
-type unAuthorizedWithoutErrorCodeTransportChecker struct {
```

Comment:
does the new version of ggcr make this obsolete?

Reply:
Sort of. We can get the status code now directly on the transport error.